dependencies.props

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    A CurrentRef is a commit on the dotnet/versions master branch. It is the single source of truth
+    for which dotnet/versions commit was last used to update the dependency.
+  -->
+  <PropertyGroup>
+    <ProdConCurrentRef>7daa2e55de668c4c8ee832059aab04917153881e</ProdConCurrentRef>
```

Comment:
For context, this points to https://github.com/dotnet/versions/blob/7daa2e55de668c4c8ee832059aab04917153881e/build-info/dotnet/product/cli/release/2.1-preview1/build.xml#L2938-L2953.